_include/Location.h

```diff
@@ -0,0 +1,71 @@
+#ifndef LOCATION_H
+#define LOCATION_H
+
+#include <ostream>
+
+/**
+ A struct representing a location in two-dimensional board.
+ The location can be added to or incremented with another location,
+ and can be compared to other locations for equality or relative ordering.
+ */
```

Comment:
why you don't have a cpp file ??

Reply:
I separated the struct to a header and cpp file